### PR TITLE
Fixed memory leak

### LINKED ISSUE #198: Webots crash
**Steps to Reproduce**
Open the world `linear_motor.wbt` with a `textureQuality` option set to 1.

**System**
 - Operating System: Windows 10

**Additional context**
gdb stack:
```
Thread 1 "MainThrd" received signal SIGSEGV, Segmentation fault.
0x00007ff8904a4400 in msvcrt!memmove () from C:\WINDOWS\System32\msvcrt.dll
(gdb) #0  0x00007ff8904a4400 in msvcrt!memmove ()
   from C:\WINDOWS\System32\msvcrt.dll
#1  0x0000000066802b99 in QImage::copy(QRect const&) const ()
   from D:\msys64\mingw64\bin\Qt5Gui.dll
#2  0x0000000066803007 in QImage::detach() ()
   from D:\msys64\mingw64\bin\Qt5Gui.dll
#3  0x0000000066803a54 in QImage::bits() ()
   from D:\msys64\mingw64\bin\Qt5Gui.dll
#4  0x00000000004c441a in WbImageTexture::updateWrenTexture (this=0x296996e0)
    at nodes/WbImageTexture.cpp:156
#5  0x00000000004c496f in WbImageTexture::updateUrl (this=0x296996e0)
    at nodes/WbImageTexture.cpp:190
#6  0x00000000004c38af in WbImageTexture::preFinalize (this=0x296996e0)
    at nodes/WbImageTexture.cpp:76
#7  0x0000000000520e7a in WbPbrAppearance::preFinalize (this=0x295f8070)
    at nodes/WbPbrAppearance.cpp:87
#8  0x0000000000577de9 in WbShape::preFinalize (this=0x295adb50)
    at nodes/WbShape.cpp:69
#9  0x00000000004b0059 in WbGroup::preFinalize (this=0x295ae610)
    at nodes/WbGroup.cpp:67
#10 0x00000000005d32c8 in WbTransform::preFinalize (this=0x295ae610)
    at nodes/WbTransform.cpp:56
#11 0x0000000000591313 in WbSolid::preFinalize (this=0x295ae610)
    at nodes/WbSolid.cpp:271
#12 0x00000000004b0059 in WbGroup::preFinalize (this=0x295ac900)
    at nodes/WbGroup.cpp:67
#13 0x00000000005d32c8 in WbTransform::preFinalize (this=0x295ac900)
    at nodes/WbTransform.cpp:56
#14 0x0000000000591313 in WbSolid::preFinalize (this=0x295ac900)
    at nodes/WbSolid.cpp:271
#15 0x00000000004b0059 in WbGroup::preFinalize (this=0x294fb6d0)
    at nodes/WbGroup.cpp:67
#16 0x000000000042d07d in WbBaseNode::finalize (this=0x294fb6d0)
    at nodes/WbBaseNode.cpp:83
#17 0x0000000000583b33 in WbSimulationWorld::WbSimulationWorld (
    this=0x142a1970, protos=0x1e5a1860, tokenizer=0xd78f930)
    at engine/WbSimulationWorld.cpp:86
#18 0x000000000045430e in WbControlledWorld::WbControlledWorld (
    this=0x142a1970, protos=0x1e5a1860, tokenizer=0xd78f930)
    at control/WbControlledWorld.cpp:41
#19 0x0000000000428c32 in WbApplication::loadWorld (this=0x14293170,
    worldName=..., reloading=false) at app/WbApplication.cpp:323
#20 0x00000000004ec440 in WbMainWindow::loadWorld (this=0x142937e0,
    fileName=..., reloading=false) at gui/WbMainWindow.cpp:1256
#21 0x00000000004b529e in WbGuiApplication::loadInitialWorld (this=0xd78fd20)
    at gui/WbGuiApplication.cpp:449
#22 0x00000000004b39cf in WbGuiApplication::exec (this=0xd78fd20)
    at gui/WbGuiApplication.cpp:239
#23 0x00000000006133f9 in main (argc=1, argv=0x1422ef10) at gui/main.cpp:199
```

### PR DESCRIPTION
**Pull request by omichel**
Fixes #198.

### CODE REVIEW

**Comment by omichel** [comment]
test me

File: src/webots/nodes/WbImageTexture.cpp [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#include <QtCore/QDebug>
```
Comment (fabienrohrer) [inline src/webots/nodes/WbImageTexture.cpp]:
```suggestion
```

**Comment by omichel** [comment]
test me

**Comment by omichel** [comment]
👍

**Comment by omichel** [comment]
Thank you for the tests!

**Pull request merged**